pkg/sql/executor_ir.go

```diff
@@ -93,6 +94,33 @@ func RunSQLProgram(sqlProgram string, db *DB, modelDir string, session *pb.Sessi
 	return rd
 }
 
+// ParseSQLStatement parse the input SQL statement and output IR in probobuf format
+func ParseSQLStatement(sqlProgram string, session *pb.Session) (string, error) {
```

Comment:
Please rename `sqlProgram` to `sql` since IR generation doesn't work on multiple statements due to 

https://github.com/sql-machine-learning/sqlflow/blob/91d94030ffe9f62d8f9af8648d39d25a9737c4d1/pkg/sql/executor_ir.go#L142-L149